libs/editor/WordPressEditor/src/main/res/layout/fragment_gutenberg_editor.xml

```diff
@@ -6,14 +6,6 @@
     android:layout_height="match_parent"
     android:layout_width="match_parent">
 
-    <org.wordpress.aztec.toolbar.AztecToolbar
-        android:id="@+id/formatting_toolbar"
```

Comment:
It seems we can remove the `layout_above` from the LinearLayout below too. While at it, it seems that the root RelativeLayout has no other children than the LinearLayout itself so, wdyt about removing one or the other?

Reply:
That's right! removed the now unneeded wrapping `RelativeLayout` in afcf44a581